en/lessons/basics/enum.md

```diff
@@ -195,11 +195,11 @@ iex> Enum.sort([%{:count => 4}, %{:count => 1}])
 [%{count: 1}, %{count: 4}]
 ```
 
-### uniq_by
```

Comment:
Why don't you add both?

Reply:
@kodnin I agree with @vbrazo here
Depending on the use case you might choose which function works best for you :)